### PR TITLE
Invoices list for add payments

### PR DESCRIPTION
**Pull request by shalapatil**
## Notion card
https://www.notion.so/saeloun/User-should-be-able-to-add-a-payments-entry-against-an-invoice-b0777f8eb0b54b0e87a51393bebf32bc

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

**Comment by supriya3105** [comment]
@akhilgkrishnan @harshaanNihal @keshavbiswa @gowsik-ragunath Please review this PR as soon as possible.

**Review by gowsik-ragunath — APPROVED** [review]
LGTM!

File: app/policies/payment_policy.rb [hunk 1]
```diff
@@ -8,4 +8,8 @@ def index?
   def create?
     user_owner_role? || user_admin_role?
   end
+
+  def new?
+    user_owner_role? || user_admin_role?
```
Comment (akhilgkrishnan) [inline app/policies/payment_policy.rb:13]: what about the bookkeeper role?

**Review by akhilgkrishnan — COMMENTED** [review]
@shalapatil I've added a concern

File: app/policies/payment_policy.rb (hunk 1 above)
Comment (supriya3105) [reply app/policies/payment_policy.rb:13]: Will take this as feedback in separate PR

**Comment by supriya3105** [comment]
@akhilgkrishnan Merging this PR for now.

**Pull request merged**